### PR TITLE
Use chrome and firefox in headless mode

### PR DESCRIPTION
**Pull request by BeyondEvil**
Always install latest chromedriver and geckodriver.

Run both in headless mode.

### CODE REVIEW

**Comment by BeyondEvil** [comment]
Appears that Travis won't be triggered by [draft PRs](https://travis-ci.community/t/draft-pull-requests-not-being-built/2434/13)

**Comment by BeyondEvil** [comment]
Aaaand you have to close/reopen to trigger the build. https://travis-ci.community/t/removing-draft-doesnt-trigger-travis/2715/2

File: .travis.yml [hunk 1]
```diff
@@ -60,21 +54,21 @@ sudo: required
 
 env:
   global:
-  - DISPLAY=:99.0
-  - GECKODRIVER_VERSION=0.21.0
-  - PYTEST_ADDOPTS="-m 'not (chrome or edge or safari or phantomjs)'"
+  - PYTEST_ADDOPTS="-m 'not (edge or safari or phantomjs)'"
+  - GECKO_FALLBACK_VERSION="v0.24.0"
```
Comment (davehunt) [inline .travis.yml]: Let's call this `GECKODRIVER_FALLBACK_VERSION` it's more verbose, but Gecko is the engine and has it's own version.

File: installation/geckodriver.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+set -e
+
+version=${1:-latest}
+
+install_dir="/usr/local/bin"
+base_url="https://github.com/mozilla/geckodriver/releases/download"
+
+if [[ ${version} == "latest" ]]; then
+    json=$(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest)
```
Comment (davehunt) [inline installation/geckodriver.sh]: I've played with this before and hit issues with rate limits, but if this is working then great!

Comment (BeyondEvil) [reply installation/geckodriver.sh]: I hit that too, took me hours to realize, hence the fallback 😂

**Comment by BeyondEvil** [comment]
Ping @davehunt 

I think this is ready now. :)

**Pull request merged**